add.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/bndw/pick/errors"
+	"github.com/bndw/pick/utils"
+)
+
+const (
+	passwordLength = 25
+)
+
+func addCommand(args ...string) int {
+	safe, err := loadSafe()
+	if err != nil {
+		return handleError(err)
+	}
+
+	name, username, password, errCode := parseAddArgs(args)
+	if errCode > 0 {
+		return errCode
+	}
+
+	err = safe.Add(name, username, password)
+	if _, conflict := err.(*errors.AccountExists); conflict {
+		if overwrite(name) {
+			// Remove the existing credential
+			rerr := safe.Remove(name)
+			if rerr != nil {
+				return handleError(rerr)
+			}
+
+			addCommand([]string{name, username, password}...)
+		}
+	} else if err != nil {
+		return handleError(err)
+	}
+
+	fmt.Println("Credential added")
+	return 0
+}
+
+func overwrite(name string) bool {
+	prompt := fmt.Sprintf("%s already exists, overwrite (y/n) ?\n", name)
+	return utils.GetAnswer(prompt)
+}
+
+func parseAddArgs(args []string) (name, username, password string, errCode int) {
+	if len(args) > 3 {
+		fmt.Println("Usage: add [name] [username] [password]")
```

Comment:
You should remove the password argument. You probably don't want the password stored in plaintext in your .bash_history ;)